config/constants.yml

```diff
@@ -108,3 +108,51 @@ map:
   center:
     lng: 2.52
     lat: 52.41
+
+menu:
+  - sectionName : _('Qwant is also')
+    icon: hash
+    links:
+    - name : _('Qwant Music')
+      uri : https://www.qwant.com/music/
+    - name: _('Qwant boards')
+      uri : https://boards.qwant.com/
+    - name: _('Qwant on Android')
+      uri : https://play.google.com/store/apps/details?id=com.qwant.liberty
+    - name: _('Qwant on iOS')
+      uri : https://itunes.apple.com/app/qwant/id924470452
+    - name: _('Qwant Junior')
+      uri : https://www.qwantjunior.com/
+    - name: _('Qwant Lite')
+      uri : https://lite.qwant.com/
+  - sectionName : _('About us')
+    icon: user
+    links:
+    - name: _('Blog')
+      uri : https://blog.qwant.com/fr/
+    - name: _('Press')
+      uri : https://about.qwant.com/press/
+    - name: _('Jobs')
+      uri : https://about.qwant.com/jobs/
+    - name: _('F.A.Q.')
+      uri : https://help.qwant.com/
+    - name: _('Legal notices')
+      uri : https://about.qwant.com/legal/legal-notices/
+    - name: _('Terms of Service')
+      uri : https://about.qwant.com/legal/terms-of-service/qwant-search/ #TODO update to maps
```

Comment:
will not that URL depend on the user language ? (eg https://about.qwant.com/fr/legal/cgu/qwant-maps/)

Reply:
i'll keep url untranslated for now & just add maps term of service